cohortextractor/pandas_utils.py

```diff
@@ -13,7 +13,8 @@ def dataframe_to_file(df, filename):
     if filename.endswith(".csv") or filename.endswith(".csv.gz"):
         df.to_csv(filename, index=False)
     elif filename.endswith(".feather"):
-        df.to_feather(filename)
+        # zstd is a better option that the default lz4 compression
```

Comment:
Sounds sensible. Have we confirmed that our default R image can read these?

Reply:
Yes